### ISSUE TITLE
element-newline doesn't allow text on multiple lines

### ISSUE THREAD

**Issue by zacharylund** [yeonjuan/html-eslint#289]
With the config:

```json
{
  "rules": {
    "@html-eslint/element-newline": "error"
  }
}
```

this block:

```html
<div>
  Foo
  Bar
  <div>
    Baz
  </div>
</div>
```

produces the error:

```
4:3 - There should be a linebreak after element.( [@html-eslint/element-newline](https://html-eslint.org/docs/rules/element-newline) )
```

The only way to fix is to put foo and bar on the same line:

```html
<div>
  Foo Bar
  <div>
    Baz
  </div>
</div>
```

See: https://html-eslint.org/playground#eyJjb2RlIjoiPGRpdj5cbiAgRm9vXG4gIEJhclxuICA8ZGl2PlxuICAgIEJhelxuICA8L2Rpdj5cbjwvZGl2PiIsImNvbmZpZyI6eyJydWxlcyI6eyJAaHRtbC1lc2xpbnQvZWxlbWVudC1uZXdsaW5lIjoiZXJyb3IifX0sImxhbmd1YWdlIjoiaHRtbCJ9

**Comment by yeonjuan** [comment]
@zacharylund Thank you for your report. It's definitely a bug. Let me fix it

**Comment by zacharylund** [comment]
Thank you! That fixed it!
